### PR TITLE
Fixup testing a bit, add sanity test, and add testing globals to eslint

### PR DESCRIPTION
**Pull request by komali2**
Just a couple little cleany uppy things

Testing works now on `npm run test`, before it didn't. Awesomely, it seems to have a watch set up, that automatically run tests on updating code/tests. 

Also fixed a little bug.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (EddyIonescu) [inline .vscode/settings.json]: remember to add this to the gitignore :)

File: src/Map.jsx [hunk 2]
```diff
@@ -159,7 +159,7 @@ class Map extends Component {
               route={route}
               label={route.properties.name}
               handleCheckboxChange={checkedRoute => this.filterRoutes(checkedRoute)}
-              key={route.properties.name}
+              key={route.id}
```
Comment (EddyIonescu) [inline src/Map.jsx]: ++ thanks for noticing that (and gets rid of some warnings too)

File: .vscode/settings.json (hunk 1 above)
Comment (komali2) [reply .vscode/settings.json]: this should be in there in one of the commits

Comment (komali2) [reply .vscode/settings.json]: wait lol

**Pull request merged**